src/lib/components/Radio/Radio.jsx

```diff
@@ -134,6 +137,8 @@ Radio.propTypes = {
    * * `<ID>__item__<VALUE>`
    * * `<ID>__item__<VALUE>__label`
    * * `<ID>__item__<VALUE>__labelText`
+   *
+   * If `key` in the option definition object is set, then value of the `key` attribute is used instead of `<VALUE>`.
```

Comment:
I would make it clearer by changing it to:

>  If `key` in the option definition object is set, then  `option.key` is used instead of `option.value` in place of `<VALUE>`.